tests/pytests/integration/states/test_npm.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+
+
+@pytest.fixture(scope="package", autouse=True)
+def install_npm(salt_cli, salt_minion):
+    result = salt_cli.run("pkg.install", "npm", minion_tgt=salt_minion.id)
```

Comment:
Shouldn't we only be installing npm in salt-jenkins?  I feel like it would create flaky tests to install npm during the test.

Reply:
That's a good question - and normally it would but in this case checking the result of the install will just skip the tests. Worst case, all of the platforms fail to install npm and it just ends out getting skipped on every platform.

Reply:
Oh, also -- I'm also in support of installing npm on salt-jenkins, in which case this becomes a no-op.